mkosi/types.py

```diff
@@ -19,4 +19,6 @@
 _FILE = Union[None, int, IO[Any]]
 PathString = Union[Path, str]
 
-CommandArgument = Union[str, Path]
+CommandArgument = Union[str, Path, int]
+CommandLine = list[CommandArgument]
+CommandLineReadOnly = Sequence[CommandArgument]
```

Comment:
I'd prefer not to introduce these two, I feel they obfuscate what the type really is without adding much value

Reply:
My thinking was, that this actually makes this clearer (and shorter), because it clearly communicates what functions return and take, e.g. something is not just a list of strings or a list of paths or an atom of some command line, but this is explicitly a command line being returned and this is a command line being used without changing it, without having to be too deeply into Python typing and knowing the differences between `list`, `Iterable` and `Sequence`.